code/game/objects/items/weapons/clown_items.dm

```diff
@@ -214,21 +217,17 @@
 
 /obj/item/weapon/glue/update_icon()
 	..()
-	icon_state = "glue[spent]"
+	icon_state = "glue[uses]"
```

Comment:
do this as `[uses ? "1" : "0"]` in case someone badmins extra uses onto the perma glue